Lombiq.OrchardCoreApiClient.Tester/Program.cs

```diff
@@ -11,10 +10,6 @@ public static class Program
     private const string ClientId = "Console";
     private const string ClientSecret = "Password";
 
-    [SuppressMessage(
-        "Design",
-        "CA1303:Do not pass literals as localized parameters",
-        Justification = "It is not a localization issue")]
```

Comment:
Why is this better duplicated in two pragmas?

Reply:
It is not. It was again a bug where this SupressMessage was marked as remove unneccessary comment. I problem removed it and later on produced CA1303 in the files below and I supressed them because I forgot that I removed the main supression.